packages/jest-config/src/readConfigFileAndSetRootDir.ts

```diff
@@ -39,7 +39,12 @@ export default async function readConfigFileAndSetRootDir(
   let configObject;
 
   try {
-    if (isTS) {
+    if (
+      isTS &&
+      // type assertion can be removed once @types/node is updated
+      // https://nodejs.org/api/process.html#processfeaturestypescript
+      !(process.features as {typescript?: boolean | string}).typescript
```

Comment:
I can flip the order of the lines here to make codecov happy, but this seems to be a false alarm - this line is just not covered if you don't run the TS tests o_O

Reply:
Rewriting slightly to get around this.